aspnetcore/release-notes/aspnetcore-3.0.md

```diff
@@ -0,0 +1,478 @@
+---
+title: What's new in ASP.NET Core 3.0
+author: rick-anderson
+description: Learn about the new features in ASP.NET Core 3.0.
+ms.author: riande
+ms.custom: mvc
+ms.date: 09/17/2019
+uid: aspnetcore-3.0
+---
+# What's new in ASP.NET Core 3.0
+
+This article highlights the most significant changes in ASP.NET Core 3.0 with links to relevant documentation.
+
+## ASP.NET Core 3.0 only runs on .NET Core 3.0
```

Comment:
Is this really what we want to open with? Having the first thing in this doc be a feature cut seems like a bit of a downer.

Reply:
Moved to the end. I think most folks know this already.